docs/src/integrate/nodejs-api.md

```diff
@@ -601,8 +603,6 @@ const suppressedMessages = linter.getSuppressedMessages();
 console.log(suppressedMessages[0].suppressions); // [{ "kind": "directive", "justification": "Need to suppress" }]
 ```
 
-Linting message objects have a deprecated `source` property. This property **will be removed** from linting messages in an upcoming breaking release. If you depend on this property, you should now use the `SourceCode` instance provided by the linter.
```

Comment:
This property was removed in ESLint v5 (https://github.com/eslint/eslint/commit/27e3f24d88a405bb64531d26d64eb356c51f3f2b)